### ISSUE TITLE
it is possible to play video from my android phone using this player???

### ISSUE THREAD

**Issue by yogeshmoradiya121** [TheWidlarzGroup/react-native-video#603]

**Comment by radko93** [comment]
Is it possible to read readmes from your android phone?

**Comment by valerybodak** [comment]
@radko93 
Hi, could you clarify it for me in this topic https://github.com/react-native-community/react-native-video/issues/644

Thanks a lot!

**Comment by itinance** [comment]
The README only tells us how to play videos from bundled assets with "require()". I tried to play a video file that was downloaded from the Internet with [react-native-fs](https://github.com/itinance/react-native-fs) into the application caches directory. 
While it works pretty fine on iOS, it constantly fails on Android.
I've used the plain path as is ("/data/data/..../file.mp4") as well as with the prefix "file:///data/data...."

While debugging the native android code i found that "mPlayer.prepare()" will raise an exception "Prepare failed: status 0x1"

On Stackoverflow is an [interesting answer for that issue](https://stackoverflow.com/questions/3761305/android-mediaplayer-throwing-prepare-failed-status-0x1-on-2-1-works-on-2-2) stating that it will fail on missing permissions because the MediaPlayer for Android comes from another application that is not allowed to access the folders of our applications.
Maybe i could give some hints. I will investigate it further, but i am not that android-guru. Would be fine if others would join.
